src/math/Vector2.js

```diff
@@ -380,9 +380,7 @@ Object.assign( Vector2.prototype, {
 
 		// computes the angle in radians with respect to the positive x-axis
 
-		var angle = Math.atan2( this.y, this.x );
-
-		if ( angle < 0 ) angle += 2 * Math.PI;
+		var angle = Math.atan2( - this.y, - this.x ) + Math.PI;
```

Comment:
> I'm not sure whether the performance is better.

Please do only make such PRs if you can actually demonstrate a performance improvement. The problem is that the readabilty of `angle()` is now worse than before. So I don't see a fair tradeoff so far.